egs/callhome_diarization/v1/run.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+# Copyright 2017  David Snyder
+#           2017  Matthew Maciejewski
+# Apache 2.0.
+#
+# This is still a work in progress, but implements something similar to
+# Greg Sell's and Daniel Garcia-Romero's iVector-based diarization system
+# in https://www.dropbox.com/s/bj5bc6brtzt52u4/slt_gks_dgr.pdf?dl=0 .
+# The main difference is that we haven't implemented the VB resegmentation
+# yet.
+
+. cmd.sh
+. path.sh
+set -e
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+num_components=2048
+ivector_dim=128
+
+# Prepare a collection of NIST SRE data. This will be used to train the UBM,
+# iVector extractor and PLDA model.
+local/make_sre.sh data
```

Comment:
If you change the make_sre.sh to work with a data_root, you'll also need to change it here:
`data_root=/export/corpora/LDC`
`local/make_sre.sh $data_root data/`